ios/FileUtils.m

```diff
@@ -51,10 +53,12 @@ @implementation FileUtils
 /**
  * Gets the MIME type of the file from the passed in URL. The file passed in can be a video or image file format.
  * @param path - The video or image file path to get the MIME type of.
+ * @param type - Either 'video' or 'image' so the method knows how to process the media file.
  * @returns The MIME type string of the file from the passed URL.
  */
 RCT_EXPORT_METHOD(
                   getMimeType:(NSString *)path
+                  fileType:(NSString *)type
```

Comment:
Hmm this may be an issue, because we technically don't know the type of the file at this point, that is why we have the `getMimeType` method, so we can determine the type of the file. Any ideas?

Reply:
Got rid of file type argument here. It did allow us to get a better reading for video based on contents, but it's not really necessary.